app/controllers/api/v1/errors_controller.rb

```diff
@@ -31,6 +31,15 @@ def notify_subscribers
 		end
 	end
 
+    def add_error
```

Comment:
daca ai adaugat metoda asta te rog sa modifici specs sa fie pentru ea. In momentul asta ele dau fail deci trebuie reparate!!!